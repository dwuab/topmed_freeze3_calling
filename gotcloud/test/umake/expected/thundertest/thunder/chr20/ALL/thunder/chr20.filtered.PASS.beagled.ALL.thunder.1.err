
Available Options
   Shotgun Sequences : --shotgun [<outdir_path>/thunder/chr20/ALL/split/chr20.filtered.PASS.beagled.ALL.split.1.vcf.gz],
                       --maxPhred [255]
      Optional Files : --crossoverMap [], --errorMap [], --physicalMap []
      Markov Sampler : --seed [123456], --burnin, --rounds [10]
          Haplotyper : --states [400], --errorRate [0.01], --transRate [0.01],
                       --weightedStates [300], --compact, --fixTrans
             Phasing : --randomPhase, --inputPhased [ON], --refPhased
          Imputation : --geno, --quality, --dosage [ON], --probs, --mle
        Output Files : --prefix [<outdir_path>/thunder/chr20/ALL/thunder/chr20.filtered.PASS.beagled.ALL.thunder.1],
                       --phase [ON], --uncompressed, --mldetails
      Interim Output : --sampleInterval, --interimInterval

Outputing VCF file <outdir_path>/thunder/chr20/ALL/thunder/chr20.filtered.PASS.beagled.ALL.thunder.1.vcf.gz
