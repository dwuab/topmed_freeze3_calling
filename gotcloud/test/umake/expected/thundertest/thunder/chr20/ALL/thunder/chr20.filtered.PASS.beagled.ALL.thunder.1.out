Thunder_Glf 1.0.9 -- Markov Chain Haplotyping for Shotgun Sequence Data
(c) 2005-2007 Goncalo Abecasis, Yun Li, with thanks to Paul Scheet

starting LoadShotgunSamples

Loaded 60 individuals from shotgun sequence data

Loaded information on 1003 polymorphic sites


Processing input files and allocating memory for haplotyping
               Haplotyping engine (max) ... 28.2 mb
starting LoadShotgunResults

starting LoadPolymorphicSites

            Haplotyping engine (actual) ... 28.1 mb
                      Consensus builder ... 266.5 kb
                      Dosage Calculator ... 118.5 kb
Memory allocated successfully

Warning: crossover rate map file [] not available
Warning: error rate map file [] not available
Loading phased information from the input VCF file

starting LoadHaplotypesFromVCF
Found initial haplotype set

Markov Chain iteration 1 [600 mosaic crossovers]
Markov Chain iteration 2 [474 mosaic crossovers]
Markov Chain iteration 3 [415 mosaic crossovers]
Markov Chain iteration 4 [377 mosaic crossovers]
Markov Chain iteration 5 [378 mosaic crossovers]
Markov Chain iteration 6 [360 mosaic crossovers]
Markov Chain iteration 7 [337 mosaic crossovers]
Markov Chain iteration 8 [334 mosaic crossovers]
Markov Chain iteration 9 [324 mosaic crossovers]
Markov Chain iteration 10 [327 mosaic crossovers]

Merged sampled haplotypes to generate consensus
   Consensus estimated to have 549.4 errors in missing genotypes and 175.0 flips in haplotypes

Estimated mismatch rate in Markov model is: 0.00112
